sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/ThriftServerQueryTestSuite.scala

```diff
@@ -93,7 +93,10 @@ class ThriftServerQueryTestSuite extends SQLQueryTestSuite {
     "subquery/in-subquery/in-group-by.sql",
     "subquery/in-subquery/simple-in.sql",
     "subquery/in-subquery/in-order-by.sql",
-    "subquery/in-subquery/in-set-operations.sql"
+    "subquery/in-subquery/in-set-operations.sql",
+    // SPARK-29783: need to set conf
```

Comment:
Was it only because we couldn't set the configurations? If Thrift server does not respect configuration like:

```
--SET spark.sql.intervalOutputStyle = ISO_8601
```

it looks a bug to me. @wangyum

Reply:
We can remove this config:
https://github.com/apache/spark/blob/3163b6b43b99ca02642cf935d885ed2d0f98d633/sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/ThriftServerQueryTestSuite.scala#L78